contrib/ovirt/lib/ovirtlago/testlib.py

```diff
@@ -138,7 +138,7 @@ def addError(self, test, err):
     def addFailure(self, test, err):
         self._addFault(test, err)
 
-    def _addFault(self, test, err):
```

Comment:
Yes, I thought it was a typo, then noticed it was simply not used, so deleted it. I kinda twisted the term 'cleanup' here ;-)